qa/L0_trace/test.sh

```diff
@@ -188,6 +188,9 @@ fi
 if [ `grep -c "\"trace_file\":\"trace_off_to_min.log\"" ./curl.out` != "1" ]; then
     RET=1
 fi
+if [ `grep -c "\"trace_mode\":\"TRITON\"" ./curl.out` != "1" ]; then
```

Comment:
note these tests will be expecting "triton" instead "opentelemetry" after your lowercase changes